### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by Iron-Carbon** [erikwijmans/Pointnet2_PyTorch#54]
Hi Erik, thanks for this amazing pytorch implementation.
I'm a beginner, and I can't really find out where we get the validation set in your code,
since I have spilt my own validation set when training on my own dataset, I would like to know that.  
Could you please point it out for me, please. Thanks a lot!
